ppr-ui/public/config/configuration.json

```diff
@@ -1,5 +1,5 @@
 {
-  "REGISTRY_URL": "https://dev.bcregistry.ca/business/auth/",
+  "REGISTRY_URL": "https://yfthig-dev.web.app/dashboard",
```

Comment:
I assume the config variables in each environment will need updating also...

Reply:
Yes, it is in 1pass.

Reply:
Ok. (I do not have permissions to resolve this conversation.)